asdf/asdf.py

```diff
@@ -74,7 +74,7 @@ def __init__(
             The URI for this ASDF file.  Used to resolve relative
             references against.  If not provided, will be
             automatically determined from the associated file object,
-            if possible and if created from `AsdfFile.open`.
+            if possible and if created from `asdf.AsdfFile.open`.
```

Comment:
it's really neat how this does linking in the built docs!

Reply:
This is using [`automodapi`](https://github.com/astropy/sphinx-automodapi).